File IO - Stop Listening does not work for a specific folder

* :white_check_mark: **WORKS:** Stop All Listeners
* :no_entry: **BROKEN:** Stop Listening

```
[ERROR] 2023-10-26T00:31:11,334 @ qz.ws.PrintSocketClient:180
	FileIO exception occurred
java.nio.file.AccessDeniedException: File requests is not verified
	at qz.utils.FileUtilities.checkFileRequest(FileUtilities.java:243) ~[qz-tray.jar:?]
	at qz.utils.FileUtilities.getAbsolutePath(FileUtilities.java:264) ~[qz-tray.jar:?]
	at qz.utils.FileUtilities.getAbsolutePath(FileUtilities.java:256) ~[qz-tray.jar:?]
	at qz.ws.PrintSocketClient.processMessage(PrintSocketClient.java:547) ~[qz-tray.jar:?]
	at qz.ws.PrintSocketClient.lambda$onMessage$0(PrintSocketClient.java:169) ~[qz-tray.jar:?]
	at java.lang.Thread.run(Unknown Source) [?:?]

```

--- a/src/qz/ws/PrintSocketClient.java
+++ b/src/qz/ws/PrintSocketClient.java
@@ -540,6 +540,8 @@ public class PrintSocketClient {
                 break;
             }
             case FILE_STOP_LISTENING: {
+                // Coerce to trusted state for unsigned request
+                request.setStatus(RequestState.Validity.TRUSTED);
                 if (params.isNull("path")) {
                     connection.removeAllFileListeners();
                     sendResult(session, UID, null);

